lib/active_merchant/billing/gateways/paymill.rb

```diff
@@ -154,7 +154,7 @@ def save_card(credit_card)
         begin
           raw_response = ssl_request(:get, "#{save_card_url}?#{post_data(post)}", nil, {})
         rescue ResponseError => e
-          return Response.new(false, e.response.body, e.response.body, {})
+          return Response.new(false, e.response.body)
```

Comment:
Looked into that earlier, but noticed other gateways return with `Response.new(false, "a message")` for these cases. It requires changes to the `MockResponse` in the test too (doesn't implement `params`) and wanted to keep changes minimal for this edge case.